package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-request-context",
-  "version": "1.0.2",
+  "version": "2.0.0",
```

Comment:
Could you revert this change?

It is in conflicts with our release tool

Reply:
So we don't bump versions manually?

Reply:
we normally bump them directly on master on release. So multiple PRs can go in the same release.